.github/workflows/release.yml

```diff
@@ -0,0 +1,190 @@
+---
+name: "Release Pypi, Docker and GitHub"
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "The version to tag, without the leading 'v'. If omitted, will initiate a dry run (no uploads)."
```

Comment:
So we buildon every push, and deploy release on specific tag `vX.Y.Z` ? Any keyword to specify ?

Reply:
Yep, we have a CI for each commit/PR, and the release one on manual trigger.

Just have to trigger the CD action with the tag to create `e.g. 0.4.0 or 0.4.0alpha1` and it will deploy to Pypi, Github Release, create the Git tag, and push a Docker image to Github Registry.